README.md

```diff
@@ -63,6 +63,23 @@ Packer and cloudformation won't automatically remove old AMI versions, so this h
 done manually. AMIs can be deleted from the EC2 console by deregestering the AMI and
 deleting the related EBS snapshot.
 
+## Deploying AWS Lambda functions
+
+Lambda functions are created by the CloudFormation stack, but rerunning the stack won't updated
+the function code. Instead, functions can be updating using the `lambda-release` command:
```

Comment:
Is this something that will generally be done via a Jenkins job rather than from local command line?  If so we could say that in the README too, to make it clear.